README.md

```diff
@@ -28,9 +31,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
     - [Language & Framework](#language--framework)
     - [Utility](#utility)
     - [Testing](#testing)
-  - [Integrations](#integration-libraries)
-  - [Plugins](#webpack-plugins)
-  - [Tools](#webpack-tools)
+  - [Integration Libraries](#integration-libraries)
+  - [Webpack Plugins](#webpack-plugins)
+  - [Webpack Tools](#webpack-tools)
```

Comment:
Don't touch this

Reply:
Sorry, vscode updates readme's TOC automatically, I'll edit it via sublime.

Reply:
:+1: